rgw/v2/tests/s3_swift/test_bucket_listing.py

```diff
@@ -271,6 +271,17 @@ def test_exec(config):
                 else:
                     raise TestExecError("object listing via boto failed")
 
+        # radoslist on all buckets. BZ:https://bugzilla.redhat.com/show_bug.cgi?id=1892265
+        if config.test_ops.get("radoslist_all", False):
```

Comment:
I think we can remove `False` here since the default value is already set as `False` from config class